src/main/java/core/basesyntax/Utilities.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.util.Date;
+
+public interface Utilities extends Texts {
```

Comment:
What means Utilities? This is too general name. Consider removing this interface